coffee_roaster_market/apps/coffee/models.py

```diff
@@ -1,3 +1,81 @@
-from django.db import models  # noqa: F401
+from django.db import models
 
-# Create your models here.
+
+class SensorialProfileModel(models.Model):
+    accidity = models.FloatField()
+    balance = models.FloatField()
+    body = models.FloatField()
+    flavor = models.FloatField()
+    aftertaste = models.FloatField()
+    sweetness = models.FloatField()
+    clean_cup = models.FloatField()
+    # overall will be calculated from other fields
+
+    def overall(self) -> float:
+        return round(
```

Comment:
Maybe it would be better to do something like this:
```
        values = (
                self.accidity,
                self.balance,
                self.body,
                self.flavor,
                self.aftertaste,
                self.sweetness,
                self.clean_cup,
            )

        return round(sum(values) / len(values), ndigits=2)
```
So wo don't have to pass the divisor explicit (`7`).
Or maybe we can use `django.forms.models.model_to_dict` to get values from all fields first.
What do you think?

Reply:
Yes definitely, model to dict is what i was searching for.

Reply:
Done, I have also changed fields to be an `PositiveIntegerField` (I've saw that coffee rating is for example 96/100)